etc/kayobe/kolla/config/ironic/ironic-conductor.conf

```diff
@@ -0,0 +1,14 @@
+[pxe]
+image_cache_size = 81920
+image_cache_ttl = 10080
+
+[console]
+socat_address = {% raw %}{{ api_interface_address }}{% endraw %}
```

Comment:
Should this be the default in kolla-ansible?

Reply:
It wasn't at the time (found public_interface instead0, but Will added some patches upstream so it will be and this will be unnecessary